hyperstart.go

```diff
@@ -323,7 +323,9 @@ func (h *hyper) bindUnmountContainerRootfs(podID, cID string) error {
 func (h *hyper) bindUnmountAllRootfs(pod Pod) {
 	for _, c := range pod.containers {
 		h.bindUnmountContainerMounts(c.mounts)
-		h.bindUnmountContainerRootfs(pod.id, c.id)
+		if c.state.Fstype == "" {
+			h.bindUnmountContainerRootfs(pod.id, c.id)
```

Comment:
I have just wrapped the existing call in an if condition. This will require changes up the stack, something that I want to do in a separate PR.